matrix/lambdas/daemons/mapper.py

```diff
@@ -82,19 +78,13 @@ def _get_chunk_specs(bundle_fqids: typing.List[str]) -> typing.List[dict]:
         chunk_work_spec = []
         for bundle_fqid in bundle_fqids:
             bundle_uuid, bundle_version = bundle_fqid.split(".", 1)
-            zarr_store = DSSZarrStore(bundle_uuid,
-                                      bundle_version=bundle_version,
-                                      dss_instance=os.environ['DEPLOYMENT_STAGE'])
-
-            root = zarr.group(store=zarr_store)
-
-            rows_per_chunk = root.expression.chunks[0]
-            total_chunks = root.expression.nchunks
+            # TODO: Not this! This is taking advantage of the fact that every
+            # matrixable bundle in the DSS will at first have data for a single
+            # cell. That won't be true for too long
```

Comment:
When do we expect (10x?) bundles to be available in S3?

Reply:
In the DSS? Probably a couple weeks after the demo.

Reply:
Ya *DSS